### PR TITLE
Bugfix/api keys

### PR DESCRIPTION
**Pull request by TylerRockwell**
### Why?
https://github.com/smashingboxes/apprenticeship/tree/master/Curriculum/Backend/bookstore
Because the only trouble with the bookstore was that no one could sign up, and no one could buy books

### What Changed?
Updated app to use secrets.yml instead of environment for keys
Added ActionMailer default host
Updated invoice emails to show formatted shipping address.

### CODE REVIEW

File: taperole/hosts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Base Example
 [production]
-162.243.113.173 be_app_env=production be_app_branch=deployment
+162.243.113.173 be_app_env=production be_app_branch=bugfix/api-keys
```
Comment (dacur) [inline taperole/hosts]: hmm, not sure why this needed to change

**Comment by dacur** [comment]
looks fine to me  :+1:

File: taperole/hosts (hunk 1 above)
Comment (TylerRockwell) [inline taperole/hosts]: I was working on a different branch and needed to make sure the things I changed were working. I'll change it to master when everything's merged.

Comment (dacur) [inline taperole/hosts]: sounds good!  thanks for the explanation

**Comment by TylerRockwell** [comment]
Thanks, @dacur !

**Pull request merged**